project/Dependencies.scala

```diff
@@ -12,8 +12,8 @@ object Dependencies {
   val catsLaws = "org.typelevel" %% "cats-laws" % catsCore.revision
   val catsParse = "org.typelevel" %% "cats-parse" % "0.3.4"
   val circeConfig = "io.circe" %% "circe-config" % "0.8.0"
-  val circeGeneric = "io.circe" %% "circe-generic" % "0.14.0-M6"
-  val circeGenericExtras = "io.circe" %% "circe-generic-extras" % "0.13.0"
+  val circeGeneric = "io.circe" %% "circe-generic" % "0.14.1"
+  val circeGenericExtras = "io.circe" %% "circe-generic-extras" % circeGeneric.revision
```

Comment:
I believe circe-generic and circe-generic-extras have different release cycles. A new circe-generic release does not necessarily implies a new circe-generic-extras release.

Reply:
Thanks, fixed so.